README.md

```diff
@@ -2,24 +2,21 @@
 
 ![master](https://github.com/HSLdevcom/helmet-ui/actions/workflows/node.js.yml/badge.svg?branch=master)
 
-Desktop user interface for [Helmet 4 Model System](https://github.com/HSLdevcom/helmet-model-system) and
-[EMME - Multimodal Transport Planning Software](https://www.inrosoftware.com/en/products/emme/).
+Helmet UI is a desktop user interface for [Helmet 4 Model System](https://github.com/HSLdevcom/helmet-model-system). **Read [the documentation in Finnish](https://hsldevcom.github.io/helmet-ui/) to learn how to use Helmet UI and Helmet 4 Model System.** If you wish to develop the UI, continue below.
```

Comment:
I think it is good git practice to keep rows reasonably short.

```suggestion
Helmet UI is a desktop user interface for [Helmet 4 Model System](https://github.com/HSLdevcom/helmet-model-system). 
**Read [the documentation in Finnish](https://hsldevcom.github.io/helmet-ui/) to learn how to use Helmet UI and Helmet 4 Model System.**
 If you wish to develop the UI, continue below.
```

Reply:
Great catch, I totally forgot about the line lengths!